presentation/frontend/src/presentation-frontend/PresentationManager.ts

```diff
@@ -120,7 +125,8 @@ export class PresentationManager implements IDisposable {
       this.activeUnitSystem = props.activeUnitSystem;
     }
 
-    this._requestsHandler = props?.rpcRequestsHandler ?? new RpcRequestsHandler(props ? { clientId: props.clientId } : undefined);
+    const clientId = props?.clientId ?? Guid.createValue();

```

Comment:
Any reason we can't keep `clientId = undefined` as it was before?

Reply:
Restored as it was before.